system_baseline/internal_views/v1.py

```diff
@@ -46,3 +46,35 @@ def get_baselines_by_system_id(system_id=None):
     current_app.logger.audit(message, request=request, success=True)
 
     return [result.system_baseline_id for result in query_results]
+
+
+@metrics.baseline_delete_requests.time()
+@metrics.api_exceptions.count_exceptions()
+def delete_systems_by_ids(system_ids):
+    """
+    delete a list of systems given their system IDs as a list
+    """
+    # TODO: ensure_write()
+    # TODO: validate_uuids()
+    account_number = view_helpers.get_account_number(request)
+    try:
+        SystemBaselineMappedSystem.delete_by_system_ids(system_ids, account_number)
+    except Exception:
+        message = "Unknown error when deleting systems by ids"
+        current_app.logger.audit(message, request=request, success=False)
+        raise
+
+    message = "delete systems by ids"
+    current_app.logger.audit(message, request=request, success=True)
+
+    return "OK"
+
+
+@metrics.baseline_delete_requests.time()
+@metrics.api_exceptions.count_exceptions()
+def create_systems_deletion_request(body):
```

Comment:
why do we have this wrapper in addition to delete_systems_by_ids above?  Does adding body["system_ids"] instead of just passing the list give the "request" that you otherwise wouldn't have access to?

Reply:
DELETE request uses only limited url for passing system ids. POST request is not limited in this way. `connexion` with openapi spec doesn't allow us to point more endpoints to one method so we have this wrapper here.